package.json

```diff
@@ -4,9 +4,15 @@
   "description": "React components for govuk layouts and elements",
   "main": "index.js",
   "scripts": {
-    "test": "eslint .",
+    "lint": "eslint .",
+    "test": "npm run lint && npm run test:unit-built",
+    "test:unit-built": "jest --config ./scripts/jest/config.build.js --env=jsdom",
+    "test:unit-source": "jest --config ./scripts/jest/config.source.js --env=jsdom",
+    "test:watch": "npm run test:unit-source -- --watch",
+    "test:update": "npm run test -- --updateSnapshot",
     "build": "npm run build:js && npm run build:css",
-    "build:js": "babel ./src --out-dir ./components ",
+    "postbuild": "npm run test",
```

Comment:
I'm not super thrilled with the idea of re-testing on _every_ build as it's going to be the kind of thing that's done _a lot_ during development, and so having to wait the extra 10 seconds on every build is likely to get annoying quickly.

I think I'd probably prefer to invert this order of control and run `npm run build` as a `pretest` script.

Reply:
That said, I'm going to merge now, and if it gets horrific later it can change.